src/Components/test/testassets/ComponentsApp.App/Pages/FetchData.cshtml

```diff
@@ -2,7 +2,7 @@
 @page "/fetchdata/{StartDate:datetime}"
 @inject WeatherForecastService ForecastService
 
-<h1>Weather forecast</h1>
+<h1 id="fetch-data">Weather forecast</h1>
```

Comment:
Shouldn't be needed

Reply:
I've explained my motives above. Is there any specific reason why you think this is an issue? I believe that in principle, being more specific with these tests is better, but I'm all ears.

Reply:
Let's be sure we understand the real cause of the issue before we know what the principles are :)